File: 01. Data Preprocessing/data_preprocessing_template.py
# Data Preprocessing

# Importing the libraries
import numpy as np 
import matplotlib.pyplot as plt
import pandas as pd


# Import the dataset
dataset = pd.read_csv('Data.csv')
x = dataset.iloc[:, :-1].values
y = dataset.iloc[:, 3].values


# Splitting the dataset into the Training set and Test set
from sklearn.model_selection import train_test_split
xTrain, xTest, yTrain, yTest = train_test_split(x, y, test_size = 0.2, random_state = 0)
 
# Feature Scaling
# from sklearn.preprocessing import StandardScaler
# scX = StandardScaler()
# xTrain = scX.fit_transform(xTrain)
# xTest = scX.transform(xTest)

File: 01. Data Preprocessing/data_preprocessing_template.R
# Data Preprocessing

# Import the dataset
dataset = read.csv('Data.csv')
# dataset = dataset[, 2:3]



# Splitting the dataset into the Training set and Test set
# install.packages('caTools')
library(caTools)
set.seed(123)
split = sample.split(dataset$Purchased, SplitRatio = 0.8) # TRUE -> Training set | FALSE - Test set
trainingSet = subset(dataset, split == TRUE)
testSet = subset(dataset, split == FALSE)

# Feature Scaling
# trainingSet[, 2:3] = scale(trainingSet[, 2:3])
# testSet[, 2:3] = scale(testSet[, 2:3])








